pages/common/img2pdf.md

```diff
@@ -1,24 +1,25 @@
 # img2pdf
 
-> Command-line lossless converter of raster images to PDF.
+> Losslessly convert raster images to a PDF file.
+> Some supported image formats are: GIF, JPEG, JPEG2000, PNG, GIF and TIFF.
 > More information: <https://gitlab.mister-muffin.de/josch/img2pdf>.
 
 - Convert one or more images to a single PDF, each image being on its own page:
 
-`img2pdf {{path/to/image1.jpg path/to/image2.jpg ...}} --output {{path/to/file.pdf}}`
+`img2pdf {{path/to/image1.ext path/to/image2.ext ...}} --output {{path/to/file.pdf}}`
 
 - Convert only the first frame of a multi-frame image to PDF:
 
 `img2pdf {{path/to/file.gif}} --first-frame-only --output {{path/to/file.pdf}}`
 
 - Auto orient the image, use a page size of A4 in landscape mode, and set a border of 2cm horizontally and 5.1cm vertically:
 
-`img2pdf {{path/to/file.jpg}} --auto-orient --pagesize {{A4^T}} --border {{2cm}}:{{5.1cm}} --output {{path/to/file.pdf}}`
+`img2pdf {{path/to/image.ext}} --auto-orient --pagesize A4^T --border 2cm:5.1cm --output {{path/to/file.pdf}}`
```

Comment:
In my opinion, we should change the description before marking placeholders as non-placeholders in the examples.

```suggestion
`img2pdf {{path/to/image.ext}} --auto-orient --pagesize {{A4^T}} --border {{2cm}}:{{5.1cm}} --output {{path/to/file.pdf}}`
```

Reply:
I too am in favour of a specific example, however, I think that placeholders ought to be marked as such.